code/core/src/core-server/dev-server.ts

```diff
@@ -17,19 +17,18 @@ import { getAccessControlMiddleware } from './utils/getAccessControlMiddleware';
 import { getStoryIndexGenerator } from './utils/getStoryIndexGenerator';
 import { getMiddleware } from './utils/middleware';
 import { openInBrowser } from './utils/open-in-browser';
-import { router } from './utils/router';
 import { getServerAddresses } from './utils/server-address';
 import { getServer } from './utils/server-init';
 import { useStatics } from './utils/server-statics';
 
 export async function storybookDevServer(options: Options) {
-  const app = express();
-
-  const [server, features, core] = await Promise.all([
-    getServer(app, options),
-    options.presets.apply('features'),
```

Comment:
Is `features` no longer used?

Reply:
it was never used actually, even before this PR.